wls-eai-service/src/test/resources/httpRequests/wahlvorstand.http

```diff
@@ -0,0 +1,146 @@
+POST {{ SSO_URL }}/auth/realms/wls_realm/protocol/openid-connect/token
+Content-Type: application/x-www-form-urlencoded
+
+password = test &
+grant_type = password &
+client_secret = top-secret &
+client_id = wls &
+username = wls_all
+
+> {%
+    client.global.set("auth_token", response.body.access_token);
+    client.global.set("token_type", response.body.token_type);
+%}
+
+### get userinfo with auth_token
+GET {{ SSO_URL }}/auth/realms/wls_realm/protocol/openid-connect/userinfo
+Authorization: {{ token_type }} {{ auth_token }}
+
+### GET Wahlvorstand wbz1
+GET {{ WLS_EAI_SERVICE_URL }}/wahlvorstaende?wahlbezirkID=00000000-0000-0000-0000-000000000001
+Authorization: {{ token_type }} {{ auth_token }}
+
+### GET Wahlvorstand wbz2
+GET {{ WLS_EAI_SERVICE_URL }}/wahlvorstaende?wahlbezirkID=00000000-0000-0000-0000-000000000002
+Authorization: {{ token_type }} {{ auth_token }}
+
+### GET Wahlvorstand wbz3
+GET {{ WLS_EAI_SERVICE_URL }}/wahlvorstaende?wahlbezirkID=00000000-0000-0000-0000-000000000003
+Authorization: {{ token_type }} {{ auth_token }}
+
+### GET Wahlvorstand wbz4
+GET {{ WLS_EAI_SERVICE_URL }}/wahlvorstaende?wahlbezirkID=00000000-0000-0000-0000-000000000004
+Authorization: {{ token_type }} {{ auth_token }}
+
+### GET Wahlvorstand not found
+GET {{ WLS_EAI_SERVICE_URL }}/wahlvorstaende?wahlbezirkID=00000000-9999-0000-0000-000000000000
+Authorization: {{ token_type }} {{ auth_token }}
+
+### POST Update Wahlvorstand wbz4
```

Comment:
```suggestion
### PUT Update Wahlvorstand wbz4
```

Reply:
fxied in c47f99791311e4fb4b03828d40dae4979c050130